bitnami/airflow/Chart.yaml

```diff
@@ -32,4 +32,4 @@ name: airflow
 sources:
   - https://github.com/bitnami/bitnami-docker-airflow
   - https://airflow.apache.org/
-version: 12.1.8
+version: 12.1.9
```

Comment:
Could you bump the minor version? It is adding a non-breaking feature

Reply:
@javsalgar I totally can. Just to make sure I'm clear that's 12.X right? So in this case we'd be going 12.1.8 to 12.2?

Thanks!

Reply:
Hey @javsalgar, I pushed this up just now to 12.2.0. Would you mind confirming for me that's what we're after? 

Thanks!